docs/release-notes/snapcraft-8-7.rst

```diff
@@ -42,6 +42,9 @@ Remote builds can now use the ``--build-for`` option to filter entries in an
 Support for confdbs
 ~~~~~~~~~~~~~~~~~~~
 
+Confdbs define the configuration of devices, including schemas, storage, access
```

Comment:
It would be ideal to include an external link here, but I can't find a decent overview of confdb/corosync. Do you have one handy?

Reply:
No, unfortunately it's undocumented other than this [outdated API reference](https://dashboard.snapcraft.io/docs/reference/v2/en/registries.html) and this [interface](https://snapcraft.io/docs/confdb-interface) reference.  It's why we got [this feedback](https://forum.snapcraft.io/t/call-for-testing-snapcraft-8-7-0/45762/2?u=mr_cal) and prompted me to explain the concept in the release notes.

I believe the snapd team is going to publish documentation later this cycle.

Reply:
Got it. This is fine as-is, then.